pkg/koordlet/metriccache/api.go

```diff
@@ -26,6 +26,16 @@ type CPUMetric struct {
 	CPUUsed resource.Quantity
 }
 
+type GPUMetric struct {
+	CardIdx     int32             // index starting from 0
+	ID          string            // device UUID
+	Type        string            // product brand type of GPU
+	Pids        []int32           // list of process IDs on this device
```

Comment:
detailed pids may be unnecessary to report

Reply:
agreed, removed.